src/gui/guiFormSpecMenu.cpp

```diff
@@ -3396,6 +3421,47 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 			}
 
 			m_selected_dragging = false;
+
+			// If the mouse was let go off once and we are in left drag mode
+			if (m_auto_place && m_left_dragging) {
+				int each = MYMIN(m_selected_amount_left_drag_copy/m_left_drag_to_stack.size(), m_left_drag_stack.getStackMax(m_client->idef()));
+
+				m_selected_amount = m_selected_amount_left_drag_copy;
+
+				for (std::set<ItemSpec*>::iterator it=m_left_drag_to_stack.begin(); it!=m_left_drag_to_stack.end(); ++it) {
```

Comment:
```suggestion
				for (auto &s : m_left_drag_to_stack) {
```

In addition, I see a couple of lines above 80 characters.